### PR TITLE
Assassinate mission misc fixes

### PR DESCRIPTION
**Pull request by Tiny-DM**
## What type of PR is this.
1. [X] Bug
2. [ ] Change
3. [ ] Enhancement

### What have you changed and why?
Stringtable: Changed SpecOps to Special Forces. Personal preference, I've heard this and "SF" a lot more than SpecOps, and it's a bit less confusing
Officer mission should render time and faction correctly, and removed a few useless lines
Traitor mission, removed a few useless lines and the old pvp mission
SpecOps: The faction should be displayed correctly, SEE BELOW    

### Please specify which Issue this PR Resolves.
closes https://discord.com/channels/449481990513754112/647694189236322305/1139340437132419212 

### Please verify the following and ensure all checks are completed.

1. [X] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [X] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
Officer: Move to the middle of the peninsula south of the southwest airbase, and call an assassinate mission, or move HQ to the middle of the ocean.

********************************************************
Notes:

### CODE REVIEW

File: A3A/addons/core/functions/Missions/fn_AS_Traitor.sqf [hunk 1]
```diff
@@ -57,8 +52,7 @@ _groupTraitor selectLeader _traitor;
 _posTsk = (position _houseX) getPos [random 100, random 360];
 
 private _taskId = "AS" + str A3A_taskCount;
-[[teamPlayer,civilian],_taskID,[format [localize "STR_A3A_fn_mission_as_traitor_text",_nameDest,_displayTime,FactionGet(occ,"name")],localize "STR_A3A_fn_mission_as_traitor_titel",_markerX],_posTsk,false,0,true,"Kill",true] call BIS_fnc_taskCreate;//old pvp blufor mission - won't localize
-[[Occupants],_taskID+"B",[format ["We arranged a meeting in %1 with a %3 contact who may have vital information about their Headquarters position. Protect him until %2.",_nameDest,_displayTime,FactionGet(reb,"name")],"Protect Contact",_markerX],getPos _houseX,false,0,true,"Defend",true] call BIS_fnc_taskCreate;//old pvp blufor mission - won't localize
+[[teamPlayer,civilian],_taskID,[format [localize "STR_A3A_fn_mission_as_traitor_text",_nameDest,_displayTime,FactionGet(occ,"name")],localize "STR_A3A_fn_mission_as_traitor_titel",_markerX],_posTsk,false,0,true,"Kill",true] call BIS_fnc_taskCreate;
```
Comment (jaj22) [inline A3A/addons/core/functions/Missions/fn_AS_Traitor.sqf:55]: If you remove the blufor task then you need to adjust the A3A_fnc_taskSetState calls accordingly. I believe that's the fourth parameter.

File: A3A/addons/core/Stringtable.xml [hunk 2]
```diff
@@ -3566,7 +3566,7 @@
         <Czech>Někdo našel informace o rodině zrádce. Další problémy už působit nebude.</Czech>
       </Key>
       <Key ID="STR_A3A_fn_mission_as_traitor_text">
-        <Original>A traitor has scheduled a meeting with %4 in %1. Kill him before he provides enough intel to give us trouble. Do this before %2. We don't where exactly this meeting will happen. You will recognise the building by the nearby Offroad and %3 presence.</Original>
+        <Original>A traitor has scheduled a meeting with %3 forces in %1. Kill him before he provides enough intel to give us trouble. Do this before %2. We don't know where exactly this meeting will happen. You will recognize the building by the nearby camo-painted Offroad and %3 presence.</Original>
```
Comment (jaj22) [inline A3A/addons/core/Stringtable.xml:3569]: Is it always camo-painted?

Comment (Bob-Murphy) [reply A3A/addons/core/Stringtable.xml:3569]: It's neither always an offroad nor is it camo-painted. Depends on modsets/templates.

Comment (Tiny-DM) [reply A3A/addons/core/Stringtable.xml:3569]: Got it, I'll remove it. I was hoping to help distinguish between the other parked civilian vehicles, but there aren't that many anyways.

File: A3A/addons/core/functions/Missions/fn_AS_Traitor.sqf (hunk 1 above)
Comment (Tiny-DM) [reply A3A/addons/core/functions/Missions/fn_AS_Traitor.sqf:55]: bump from Discord

Comment (jaj22) [reply A3A/addons/core/functions/Missions/fn_AS_Traitor.sqf:55]: Lines 135 and 174.

**Comment by Tiny-DM** [comment]
all issues should be fixed. ready for review

**Comment by jaj22** [comment]
You managed to test the specops mission eventually, right?

**Comment by Tiny-DM** [comment]
@jaj22 Yes. I got the file running properly with the includes, I really should have asked for clarification earlier. Sorry about that.

**Review by jaj22 — APPROVED** [review]

File: A3A/addons/core/Stringtable.xml [hunk 3]
```diff
@@ -3566,7 +3566,7 @@
         <Czech>Někdo našel informace o rodině zrádce. Další problémy už působit nebude.</Czech>
       </Key>
       <Key ID="STR_A3A_fn_mission_as_traitor_text">
-        <Original>A traitor has scheduled a meeting with %4 in %1. Kill him before he provides enough intel to give us trouble. Do this before %2. We don't where exactly this meeting will happen. You will recognise the building by the nearby Offroad and %3 presence.</Original>
+        <Original>A traitor has scheduled a meeting with %3 forces in %1. Kill him before he provides enough intel to give us trouble. Do this before %2. We don't know where exactly this meeting will happen. You will recognize the building by the nearby Offroad and %3 presence.</Original>
```
Comment (Bob-Murphy) [inline A3A/addons/core/Stringtable.xml:3569]: As stated, it ain't always an offroad like for example when playing with SOG or Spearhead.
`You will recognize the building by the nearby parked vehicle and %3 presence.` would make more sense.

**Review by Bob-Murphy — CHANGES_REQUESTED** [review]

**Comment by Tiny-DM** [comment]
Change resolved. In the future it might be cool to grab the name for the vehicle it spawns and trim it down, but that's a minor fix for the future.

**Review by Bob-Murphy — APPROVED** [review]

**Pull request merged**